### PR TITLE
Use new assembly name for coreclr unix

### PR DESCRIPTION
**Pull request by BrennanConroy**
@victorhurdugaci @moozzyk

### CODE REVIEW

File: src/dnx.coreclr.unix/dnx.coreclr.cpp [hunk 1]
```diff
@@ -261,14 +261,14 @@ extern "C" HRESULT CallApplicationMain(PCALL_APPLICATION_MAIN_DATA data)
...
-                         "dnx.coreclr.managed",
+                         "Microsoft.Dnx.Host.CoreClr",
                          sizeof(property_keys) / sizeof(property_keys[0]),
                          property_keys,
                          property_values,
                          data->argc,
                          (const char**)data->argv,
                          nullptr,
-                         "dnx.coreclr.managed, Version=0.1.0.0",
+                         "Microsoft.Dnx.Host.CoreClr, Version=0.1.0.0",
```
Comment (moozzyk) [inline src/dnx.coreclr.unix/dnx.coreclr.cpp]: nit: I changed the version to `0.0.0.0` in the windows bootstrapper. The version seems to be required even though it appears it is not being used.

**Comment by moozzyk** [comment]
:ship::it:

**Comment by victorhurdugaci** [comment]
:shipit:

**Comment by anurse** [comment]
:shipit:

**Comment by BrennanConroy** [comment]
d7e305e3e364041d4f2312d46f61317d75564ff2

**Pull request closed without merging**